app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: ENV["email"]
```

Comment:
I think we should use a more descriptive name for this Environment variable, i would say something like 'default-from-email'. What do you think @sebasoga @linatodi @angela4713?